packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -47,14 +47,16 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage>
   @override
   void initState() {
     super.initState();
-    _model = PersonalizedRankingModel(widget.barcodes);
+    _model = PersonalizedRankingModel(
+      widget.barcodes,
+      context.read<LocalDatabase>(),
+    );
   }
 
   @override
   Widget build(BuildContext context) {
     final ProductPreferences productPreferences =
         context.watch<ProductPreferences>();
-    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
```

Comment:
This leads to this app not beeing refreshed anymore, does it?

Reply:
@M123-dev You're right, it's a mistake. Fixed in the next push.